Tools/CMake/AMReX_Config.cmake

```diff
@@ -140,22 +140,25 @@ function (configure_amrex)
    endif ()
 
    if ( BUILD_SHARED_LIBS OR AMReX_CUDA )
+      set(host_link_supported OLD)
+      if(CMP0105)
+        cmake_policy(GET CMP0105 host_link_supported)
+      endif()
+
       if(APPLE)
```

Comment:
@robertmaynard in this line: should we roll back or make the `LINKER:` conditional as well?
We don't have CUDA these days on macOS anymore, but if `LINKER:` is a CMake 3.18+ feature we might better not place it in the `if(APPLE)` path unconditionally, yet.

Reply:
`LINKER:` was added at the same time as `target_link_options`, we just missed the `NVIDIA-CUDA` language/compiler configuration and so it didn't have a proper mapping.

Reply:
(answered inline below)